common/src/main/scala/uk/ac/wellcome/dynamo/package.scala

```diff
@@ -11,4 +14,13 @@ package object dynamo {
     )(
       _.toString
     )
+
+  // DynamoFormat for tagged HLists
```

Comment:
I think I know what an HList is (although a reminder would be helpful, as it seems to be a shapeless concept) – what’s a tagged HList?

Reply:
I think it's a field key and value pair.